README.md

```diff
@@ -10,17 +10,17 @@ can be found here:
 [https://indico.bnl.gov/category/443/](https://indico.bnl.gov/category/443/)
 
 A preliminary tutorial on JANA2 is being developed here:
-[https://faustus123.github.io/tutorial-jana2](https://faustus123.github.io/tutorial-jana2/index.html)
+[https://faustus123.github.io/tutorial-jana2](https://eic.github.io/tutorial-jana2/)
```

Comment:
I guess it also should be 

[https://eic.github.io/tutorial-jana2/](https://eic.github.io/tutorial-jana2/)